adam-core/src/main/java/org/bdgenomics/adam/io/InterleavedFastqInputFormat.java

```diff
@@ -127,13 +123,15 @@ public InterleavedFastqRecordReader(Configuration conf, FileSplit split) throws
          */
         private void positionAtFirstRecord(FSDataInputStream stream) throws IOException {
             Text buffer = new Text();
-            int bytesRead = 0;
-            
+
+            if (true) { // (start > 0) // use start>0 to assume that files start with valid data
```

Comment:
Just out of curiosity, why is this wrapped in "if (true)"?  Is it just because you're reverting and "this is the way it used to be?"